pkg/cmd/server/api/v1/types.go

```diff
@@ -68,6 +69,11 @@ type NodeConfig struct {
 
 	// IPTablesSyncPeriod is how often iptable rules are refreshed
 	IPTablesSyncPeriod string `json:"iptablesSyncPeriod"`
+
+	// ProjectEmptyDirQuota is the resource quantity to be used as quota for emptyDir volumes.
+	// This is currently implemented only for XFS, and is a limit per namespace per node.
+	// (this may change in the future)
+	ProjectEmptyDirQuota *resource.Quantity `json:"projectEmptyDirQuota"`
```

Comment:
Upstream type being resource.Quantity? I don't think it's versioned, only hit I can find is in Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/resource/quantity.go and there don't appear to be any version sub-dirs there. It serializes to just a string so possibly deemed not required for this.